README.md

```diff
@@ -2,8 +2,8 @@
   <img alt="giskardlogo" src="https://raw.githubusercontent.com/giskard-ai/giskard/main/readme/giskard_logo.png#gh-light-mode-only">
   <img alt="giskardlogo" src="https://raw.githubusercontent.com/giskard-ai/giskard/main/readme/giskard_logo_green.png#gh-dark-mode-only">
 </p>
-<h1 align="center" weight='300' >The testing & evaluation framework for LLMs & other AI models</h1>
-<h3 align="center" weight='300' >Scan AI models to detect risks of performance issues. In 4 lines of code. </h3>
+<h1 align="center" weight='300' >The Evaluation & Testing framework for LLMs & ML models</h1>
+<h3 align="center" weight='300' >Control risks of performance issues, biases and security vulnerabilities in AI models</h3>
```

Comment:
Security vulnerabilities is slightly misleading since we don't cover deserialization attacks, code exec, etc

Reply:
Yeah good idea, ill change it

Reply:
"Control risks of performance, biases and security issues in AI models"